src/bun.js/api/JSBundler.zig

```diff
@@ -799,6 +803,11 @@ pub const JSBundler = struct {
         /// Faster path: skip the extra threadpool dispatch when the file is not found
         was_file: bool = false,
 
+        // We only allow the user to call defer once right now
+        called_defer: bool = false,
+
+        const debug_deferred = bun.Output.scoped(.BUNDLER_DEFERRED, true);
```

Comment:
the log scope name is misleading? it's not specific to deferred (deinit at the bottom)